django/core/management/commands/makemigrations.py

```diff
@@ -194,13 +194,17 @@ def handle_merge(self, loader, conflicts):
             questioner = InteractiveMigrationQuestioner()
         else:
             questioner = MigrationQuestioner(defaults={'ask_merge': True})
+
         for app_label, migration_names in conflicts.items():
             # Grab out the migrations in question, and work out their
             # common ancestor.
             merge_migrations = []
             for migration_name in migration_names:
                 migration = loader.get_migration(app_label, migration_name)
-                migration.ancestry = loader.graph.forwards_plan((app_label, migration_name))
+                migration.ancestry = list(filter(
```

Comment:
Any reason not to use a list comprehension? I think that's usually more readable.